Tasks/QuickPerfTest/CltThresholdValidationHelper.ps1

```diff
@@ -0,0 +1,113 @@
+Class ThresholdRule
+{
+	[string]$counterInstanceName
+	[bool]$isHigherBetter
+	[float]$thresholdValue
+}
+
+function ValidateThresholdRule($cltAccountUrl,$headers,[ThresholdRule]$thresholdRule,$testRunId)
+{
+ $counterInstances = Get-CounterInstances $cltAccountUrl $headers $testRunId
+ return ValidateThreshold $cltAccountUrl $headers $thresholdRule $testRunId $counterInstances
+}
+
+function ValidateThreshold($cltAccountUrl,$headers,[ThresholdRule]$thresholdRule,$testRunId,$counterInstances)
+{
+ $counterInstanceId = GetCounterInstanceId $counterInstances $thresholdRule.counterInstanceName
```

Comment:
>GetCounterInstanceId [](start = 22, length = 20)

Minor: lets keep one pattern in method names either '<verb>-<resource>' or '<Verb><Resource>'